### PR TITLE
Add a11y_theme fields to clients daily

### PR DESCRIPTION
**Pull request by scholtzan**
For migrating accessibility dashboards to Looker: https://mozilla-hub.atlassian.net/browse/DENG-52

Making the fields `a11y_theme`, `a11y_hcm_foreground` and `a11y_hcm_background` available in clients_daily

### CODE REVIEW

**Comment by scholtzan** [comment]
We'd want to backfill clients_daily back to 2021-05-01

File: sql/moz-fx-data-shared-prod/telemetry_derived/addons_daily_v1/schema.yaml [hunk 1]
```diff
@@ -0,0 +1,187 @@
+fields:
```
Comment (jklukas) [inline sql/moz-fx-data-shared-prod/telemetry_derived/addons_daily_v1/schema.yaml:1]: I think that adding this schema file is an unrelated change. Can we remove this?

**Review by jklukas — APPROVED** [review]
r+wc

**Comment by scholtzan** [comment]
Table schemas have been updated.
